packages/tailwind/src/utils/get-css-for-markup.ts

```diff
@@ -1,8 +1,11 @@
-import postcss from "postcss";
 import tailwindcss from "tailwindcss";
 import type { CorePluginsConfig } from "tailwindcss/types/config";
 import postcssCssVariables from "postcss-css-variables";
 import type { TailwindConfig } from "../tailwind";
+// this is to avoid Server Action problems
+
+// eslint-disable-next-line @typescript-eslint/consistent-type-imports, @typescript-eslint/no-var-requires
```

Comment:
```suggestion

// this is to avoid Server Action problems
// eslint-disable-next-line @typescript-eslint/consistent-type-imports, @typescript-eslint/no-var-requires
```